server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js

```diff
@@ -24,7 +24,7 @@ BuildOutputObserver.prototype = {
         this.was_building = false;
         this.is_output_empty = false;
         this.is_completed = false;
-
+        this.ansi_up = ansi_up.ansi_to_html_obj();
```

Comment:
Keep state across invocations of ansi_up so that we can improve colouring across chunk boundaries.